doctr/models/recognition/core.py

```diff
@@ -80,6 +80,8 @@ def __init__(
 
         self.pre_processor = pre_processor
         self.model = model
+        self.critical_ar = 8  # Critical aspect ratio
```

Comment:
Mind adding an option to use the crop_split?
Like `use_crop_splitting` as an arg that is saved as an attribute?